### ISSUE TITLE
Slack address on accessibility home page

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-backlog#123]
From: 
Get in touch
Join us on the NHS digital service manual Slack workspace or email the standards team at service-manual@nhs.net.
To:
Get in touch
Join us on the NHS digital service manual Slack workspace [keep link] or email the service manual team at service-manual@nhs.net.

**Comment by vlj83** [comment]
Updated and sent for review

**Comment by sarawilcox** [comment]
All done.
